plugins/coverbrowser.koplugin/main.lua

```diff
@@ -159,12 +160,65 @@ function CoverBrowser:addToMainMenu(menu_items)
                         },
                     },
                 },
+                -- Misc settings
+                {
+                    text = _("Other settings"),
+                    sub_item_table = {
+                        {
+                            text = _("Show hint for books with description"),
+                            checked_func = function() return not BookInfoManager:getSetting("no_hint_description") end,
+                            callback = function()
+                                if BookInfoManager:getSetting("no_hint_description") then
+                                    BookInfoManager:saveSetting("no_hint_description", false)
+                                else
+                                    BookInfoManager:saveSetting("no_hint_description", true)
+                                end
+                                self:refreshFileManagerInstance()
+                            end,
+                        },
+                        {
+                            text = _("Show hint for opened books in History"),
```

Comment:
Thought that as the history widget is titled "History", that made it clearer it was this widget, and not just any history :)